testsuites/CBLTester/upgrade_tests/test_cbl_upgrade.py

```diff
@@ -412,3 +425,143 @@ def test_upgrade_testsever_app(params_from_base_suite_setup):
     db.create_bulk_docs(2, "cbl-upgrade-docs_new", db=source_db, channels=channels_sg)
     upgraded_cbl_doc_count = db.getCount(source_db)
     assert upgraded_cbl_doc_count > base_cbl_doc_count, " Number of docs count is not matching"
+
+
+@pytest.mark.listener
+@pytest.mark.upgrade_test
+def test_switch_dbs_with_two_cbl_platforms(params_from_base_suite_setup):
```

Comment:
Adding this test won't run as part of the regression job. You need to set up seperate jobs to run these tests again. Is it possible to have it as part of functional tests folder, otherwise it is hassle to set up new jobs just for this and parent job shows too many jobs. it is hard to go through if too many jobs. It is easy if it is part of existing jobs

Reply:
As we discussed during the daily meeting , we are going to have this way